translations/ai_rules.tex/pl.po

```diff
@@ -177,15 +177,15 @@ msgid ""
 "Shuffle the AI and Spell Decks during setup after building them.\n"
 "\n"
 msgstr ""
-"Bohaterowie AI używają dwóch Talii Kart podczas Walki: \\textbf{Talii AI} oraz \\textbf{Talii Magii}. Talia AI jest tworzona przy użyciu trzech rodzajów Kart AI: Mocy \\includesvg[height=10px]{\\svgs/might.svg}, Magii \\includesvg[height=10px]{\\svgs/magic.svg} i Zdolności \\includesvg[height=10px]{\\svgs/skill.svg}.\n"
-"Każdy Scenariusz Kampanii podaje liczbę Kart każdego rodzaju Kart AI wchodzących w skład Talii AI.Karty Mocy \\includesvg[height=10px]{\\svgs/might.svg} i Zdolności \\includesvg[height=10px]{\\svgs/skill.svg} mają dwa warianty. Wybierz te karty \\textbf{losowo} podczas budowania Talii. Jeśli w Talii są Karty Zdolności, wyszukaj Karty Zdolności, do których się odnoszą, i odłóż je na bok.\n"
+"Bohaterowie AI używają dwóch Talii Kart w Walce: \\textbf{Talii AI} oraz \\textbf{Talii Magii}. Talia AI jest tworzona z trzech rodzajów Kart AI: Mocy \\includesvg[height=10px]{\\svgs/might.svg}, Magii \\includesvg[height=10px]{\\svgs/magic.svg} i Zdolności \\includesvg[height=10px]{\\svgs/skill.svg}.\n"
```

Comment:
W przypadku AI - Skill - to Umiejętności

```suggestion
"Bohaterowie AI używają dwóch Talii Kart w Walce: \\textbf{Talii AI} oraz \\textbf{Talii Magii}. Talia AI jest tworzona z trzech rodzajów Kart AI: Mocy \\includesvg[height=10px]{\\svgs/might.svg}, Magii \\includesvg[height=10px]{\\svgs/magic.svg} i Umiejętności \\includesvg[height=10px]{\\svgs/skill.svg}.\n"
```

Reply:
A to wyszło z rozpędu wczoraj przez przypadek...